src/LiveComponent/src/Resources/views/form_theme.html.twig

```diff
@@ -0,0 +1,35 @@
+{%- block live_collection_widget -%}
```

Comment:
I think this should be a different name. Maybe `live_collection_form_theme.html.twig`?

Reply:
I think there should be only one theme per component (and per variant like e.g. bootstrap or tailwind) and not per type, and maybe later we would like to add more blocks for an another type, so I would keep the name. WDYT?